src/main/java/amazon/FashionOfWomenFromRitualPage.java

```diff
@@ -27,17 +23,12 @@ public FashionOfWomenFromRitualPage(WebDriver driver) {
     protected WebElement shopAllDealsButton;
     @FindBy(xpath = "//li[@data-page-number='1']")
     protected WebElement firstPageOfThePagination;
+    @FindBy(linkText = "Up to $25")
+    protected WebElement priceFilterInPage;
```

Comment:
What if your other task would be click on other price filter? Should you define another WebElement?
Your code must be as much general as possible, but you almost always hardcode it.